app/uk/gov/hmrc/tradergoodsprofilesrouter/models/response/GoodsItemRecords.scala

```diff
@@ -155,4 +155,21 @@ object GoodsItemRecords {
       case AccreditationStatus.Rejected => AdviceStatus.AdviceNotProvided
       case _                            => AdviceStatus.withName(accreditationStatus.entryName)
     }
+
+  private def translateReviewReason(reviewReason: Option[String], toReview: Boolean): Option[String] =
+    if (!toReview) None
+    else {
+      reviewReason.collect {
+        case "mismatch"   =>
+          "HMRC have reviewed this record. The commodity code and goods description do not match. If you want to use this record on an IMMI, you'll need to amend the commodity code and the goods description."
```

Comment:
think it would be better if we move these values to constants.

Reply:
I don't think we need to create another constant class for this